integration-tests/cucumber/cucumber.spec.js

```diff
@@ -123,112 +97,134 @@ versions.forEach(version => {
             isAgentless = reportMethod === 'agentless'
             envVars = isAgentless ? getCiVisAgentlessConfig(receiver.port) : getCiVisEvpProxyConfig(receiver.port)
           })
-          it('can run and report tests', (done) => {
```

Comment:
The changes in this file are just for running this test both in serial and parallel mode